lib/oli_web/live/delivery/student/utils.ex

```diff
@@ -1129,6 +1129,14 @@ defmodule OliWeb.Delivery.Student.Utils do
   def parse_enrollment_status(:rejected), do: "Rejected invitation"
   def parse_enrollment_status(_status), do: "Unknown"
 
+  @doc """
+  Parses the certificate status and returns a human-readable string.
+  """
+  @spec parse_certificate_status(atom()) :: String.t()
+  def parse_certificate_status(:earned), do: "Earned"
+  def parse_certificate_status(:denied), do: "Denied"
+  def parse_certificate_status(_), do: "Pending"
```

Comment:
Should we modify these to match how we render those values in the UI?

<img width="1228" height="217" alt="Image" src="https://github.com/user-attachments/assets/1718d2f2-9cdb-4def-ba49-331b86df3ff4" />


```suggestion
  @spec parse_certificate_status(atom()) :: String.t()
  def parse_certificate_status(:earned), do: "Approved"
  def parse_certificate_status(:denied), do: "Denied"
  def parse_certificate_status(_), do: "In Progress"
```